portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -313,6 +313,17 @@ definition {
 			key_text = "Publish");
 	}
 
+	macro deleteABTestACSide {
```

Comment:
Hey @manuferreira, `ABTest.macro` is the file with macros that interact with DXP. Your new macro will only delete the test locally in the AC, so I think it would be better to put it in `ACTests.macro`, which are the macros that interact with the AC

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/1a41b649755e0315d14ae5c0a87b4a36684833bf